db/migrate/20240529142659_add_acknowledgement_page_to_submission.rb

```diff
@@ -0,0 +1,5 @@
+class AddAcknowledgementPageToSubmission < ActiveRecord::Migration[6.1]
+  def change
+    add_column :submissions, :acknowledgment_page_viewed_at, :datetime
```

Comment:
Not a major issue, so this is more of a suggestion, but I think `:acknowledgment_page_submitted_at` would be better here.  Just so it's clear that the timestamps is set after submission.  I've found (particularly since returning to ETDA) that being precise with the timestamp name can help to remember exactly when it should be set.